data/common/items/items/clubs_sandwich.lua

```diff
@@ -0,0 +1,76 @@
+local item = HealItem{
+    -- Item ID (optional, defaults to path)
+    id = "clubs_sandwich",
+    -- Display name
+    name = "ClubsSandwich",
+
+    -- Item type (item, key, weapon, armor)
+    type = "item",
+    -- Item icon (for equipment)
+    icon = nil,
+
+    -- Battle description
+    effect = "Heals\nteam\n~1HP",
+    -- Shop description
+    shop = "",
+    -- Menu description
+    description = "A sandwich that can be split into 3.\nHeals ~1 HP to the team.",
+
+    -- Amount healed (HealItem variable)
+    heal_amount = 30,
+
+    -- Shop sell price
+    price = 35,
+
+    -- Consumable target mode (party, enemy, noselect, or none/nil)
+    target = nil,
+    -- Where this item can be used (world, battle, all, or none/nil)
+    usable_in = "all",
+    -- Item this item will get turned into when consumed
+    result_item = nil,
+    -- Will this item be instantly consumed in battles?
+    instant = false,
+
+    -- Equip bonuses (for weapons and armor)
+    bonuses = {
+        attack = 0,
+	},
+    -- Bonus name and icon (displayed in equip menu)
+    bonus_name = nil,
+    bonus_icon = nil,
+
+    -- Equippable characters (default true for armors, false for weapons)
+    can_equip = {},
+
+    -- Character reactions (key = party member id)
+    reactions = {
+        susie = "Quit hogging!",
+        ralsei = "(It's cut evenly...)",
+        noelle = "(Kris took two thirds of it…)"
+    },
+}
+
+if Game.chapter == 1 then
```

Comment:
use datamod for this, items should NOT have top level code.

Reply:
I used `Game.chapter` for the check because the glowshard item, which has already been present in the engine, was using this code element. I fixed all items to use datamod instead.

Reply:
the glowshard only uses it for the price, not for checks. additionally this is still top level code either way

Reply:
wait. i just realized i forgot to change the top level code in the glowshard. my bad